src/Test/OpenRiaservices.EndToEnd.Wcf.Test/TestUris.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+
+public static class TestURIs
+{
+#if SILVERLIGHT
+    public static readonly Uri RootURI = System.Windows.Browser.HtmlPage.Document.DocumentUri;
+#elif VBTests
+    public static readonly Uri RootURI = new Uri("http://localhost:60000/");
+#elif ASPNETCORE
+    public static readonly Uri RootURI = new Uri("http://localhost:5246/");
+#else
+    public static readonly Uri RootURI = new Uri("http://localhost:60002/");
+#endif
+
+    // Cities
+    public static readonly Uri Cities = new Uri(RootURI, "Cities-CityDomainService.svc");
+    //public static readonly Uri Cities_AuthRequired = new Uri(RootURI, "Cities-CityDomainService_AuthRequired.svc");
+    //public static readonly Uri Cities_RoleRequired = new Uri(RootURI, "Cities-CityDomainService_RoleRequired.svc");
+
+    // EF
+    public static readonly Uri EF_Catalog = new Uri(RootURI, "TestDomainServices-EF-Catalog.svc");
```

Comment:
I've started to move some of the tests over to relative Uri's but there are a couple of tests which might require the absolute Uri (such as when creating WebDomainClient directy of doing raw Http requests)  that would need to be updated before all uris can be made relative